### DISCUSSION TITLE
Time Zone is wrong

### DISCUSSION THREAD

**Discussion by hawswebmaster** [code4recovery/12-step-meeting-list#1297] [Q&A] [answered]
Hello all,

It looks like the time zone for all of the California meetings are set as MST. California is all PST. Is there any way to change this or get this fixed? I do not enter time zones. I am guessing it is automatic in the system by the address. Link to California meetings below. Thank you.

[](https://heroinanonymous.org/meeting-list/?region=california)

**Comment by hawswebmaster** [comment]
UPDATE!!! All meetings around the country say MST. That is a problem :(

**Comment by joshreisner** [comment, marked as answer]
hi @hawswebmaster did you recently switch to TSML UI? we recommend you switch back to the Legacy UI appearance until we get a chance to handle #930

TSML doesn't yet have the ability to add time zones to listings, but the TSML UI appearance needs one, so it inherits the timezone set in WordPress and applies that to all listings. this works ok for sites that list meetings in a single timezone, but does not work in a site like yours that lists across a variety of timezones

**Reply by hawswebmaster** [reply]
We did recently switch as it fits our layout better. I am not concerned with it even being on there. Is there a way in the code to not display it at all?

**Reply by joshreisner** [reply]
TSML UI relies on timezones heavily because the meeting list starts at "now" which is different at different times in different places. if all meetings are in MST that means that visitors from New York at 8am locally will see the list starting at 6am, and visitors from California will see meetings starting at 9am. i'd switch back for now until we get a chance to add the right support to the plugin
